src/components/ui/Navigation/Navigation.styled.js

```diff
@@ -1,12 +1,18 @@
 import styled from 'styled-components';
 
 const NavigationStyle = styled.nav`
-	height: 5rem;
+	height: 6rem;
 	width: 100%;
-	z-index: 999;
-	background-color: aqua;
-	position: fixed;
+	z-index: 100;
+	background-image: ${props =>
+		props.pathName === '/' ? 'none' : 'linear-gradient(to right, black,#000c1b)'};
+	position: sticky;
+	margin-top: 2rem;
 	bottom: 0;
+	padding-bottom: 1rem;
```

Comment:
No, I just wan't able to find out why they are not horizontally centered and tried a whole bunch of things to fix that..

Reply:
Gotcha. I went into how you're running into issues with your buttons because of the line-height/margin defined in your GlobalStyles, etc. Do you still want me to give you some cues as to how to handle that? :)